web/docs/integrations/css-frameworks.md

```diff
@@ -0,0 +1,123 @@
+---
+title: CSS Frameworks
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# CSS Frameworks
+
+## Tailwind
+
+To enable support for Tailwind in your Wasp project, you simply need to add two config files (`tailwind.config.js` and `postcss.config.js`) to the root directory. When they are present, Wasp will add the necessary NPM dependencies and copy your config files into the generated project output. You can then start adding Tailwind CSS directives to your `Main.css` file and `className`s React components.
```

Comment:
True. I mainly just said that since I was making the example in the context of a new app. I could remove it and say "to your main CSS file" if that is better.

Reply:
I think that would be good! or maybe just to `your CSS file`

Reply:
I'll do a bit of research before updating this. I think `your CSS file` would work too. For the singular vs plural, I believe we (read Tailwind) do not want users to import those directives multiple times. So depending on how the CSS files get included it may result in multiple of those. Unsure if that is truly an issue though. I will research a bit more and then update the wording. 👍🏻

Reply:
Whoops disregard my comment then, this is just coming from my ignorance about how Tailwind works! I think it is good as you have it right now then!

Reply:
No, you are fine. I think I'm the one off base here. (It is ok since neither of us are Tailwind gurus, yet :D) So they def need to have those directives anywhere they want to use Tailwind stuff, but I don't get the impression there is a danger of multiple usages, so I can just say `your CSS files`. 👍🏻